### PR TITLE
GCE scripts/Dockerfile etc. for mirroring.

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: cloud/google/testtube.pem [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEw8i8S7qiGEs9NXv0ZJFh6uuOmR2Q
+7dPprzk9XNNGkUXjzqx2SDvRfiwKYwBljfWujozHESVPQyydGaHhkaSz/g==
+-----END PUBLIC KEY-----
```
Comment (pphaneuf) [inline cloud/google/testtube.pem]: Shouldn't that one go in the `cloud/keys` directory, with the others?

File: cloud/google/start_mirror.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+           s^@@ETCD_PORT@@^4001^
+           s^@@CONTAINER_HOST@@^${MIRROR_MACHINES[$i]}^
+           s^@@TARGET_LOG_URL@@^${MIRROR_TARGET_URL}^
+           s^@@TARGET_LOG_PUBLIC_KEY@@^${MIRROR_TARGET_PUBLIC_KEY}^" \
+          < ${DIR}/mirror_container.yaml  > ${MANIFEST}.${i}
+
+  ${GCLOUD} compute instances create -q ${MIRROR_MACHINES[${i}]} \
+      --machine-type ${MIRROR_MACHINE_TYPE} \
+      --image container-vm \
+      --disk name=${MIRROR_DISKS[${i}]} mode=rw boot=no auto-delete=yes \
```
Comment (pphaneuf) [inline cloud/google/start_mirror.sh]: This probably yells at you because of the lack of commas, doesn't it?

Comment (AlCutter) [inline cloud/google/start_mirror.sh]: Nah; I've got old gcloud utils installed :)

File: cloud/google/testtube.pem (hunk 1 above)
Comment (AlCutter) [inline cloud/google/testtube.pem]: Whoops, yep.

**Comment by pphaneuf** [comment]
LGTM

**Pull request merged**